git-compat-util.h

```diff
@@ -127,7 +127,9 @@
 /* Approximation of the length of the decimal representation of this type. */
 #define decimal_length(x)	((int)(sizeof(x) * 2.56 + 0.5) + 1)
 
-#if defined(__sun__)
+#ifdef __MINGW64__
+#define _POSIX_C_SOURCE 1
+#elif defined(__sun__)
```

Comment:
Would this break compiling Git with older mingw-w64? I ask because I have to do precisely that sometimes, e.g. when building MinGit backports.

Reply:
no; I obviously tested this change is safe both with the old (from SDK 1.0.8 as released) and new (currently updated) mingw-w64 headers and forgot to mention it.

I am doing an additional test with SDK 1.0.7 (even older releases won't install because of bintray or key issues), but I would be surprised that it wouldn't work.

Reply:
Okay, thanks for confirming!